### ISSUE TITLE
Bubble scaling is not working

### ISSUE THREAD

**Issue by aradiel** [Haplo064/ChatBubbles#41]
Both Player and NPC scaling don't appear to be doing anything. I thought that they would change the size of the bubbles but nothing is changing even when putting big numbers in there.

![image](https://user-images.githubusercontent.com/22911153/166110127-3e47110d-3f97-4dd7-a900-b827f6aff86a.png)

![image](https://user-images.githubusercontent.com/22911153/166110131-a94991f7-f757-402e-8631-a0f1380ca545.png)

![image](https://user-images.githubusercontent.com/22911153/166110097-8b1df905-e1da-49c2-9083-12cc096dee77.png)

![image](https://user-images.githubusercontent.com/22911153/166110070-d902f49d-5096-4a9d-90c3-d3d291825d41.png)

**Comment by Haplo064** [comment]
I'll add some debugging and push an update to assist with this, as I cannot reproduce it on my end.

**Comment by Haplo064** [comment]
Added debugging to assist with this.
If you turn on `Debug Logging` in the bubble options, you should now see lines such as:
![image](https://user-images.githubusercontent.com/11189337/166852206-efd92b25-a5d6-46e3-ae48-0399f1728547.png)
In the dalamud.log file. (Either the raw text file or by using `/xldev` and using the in-game log:
![image](https://user-images.githubusercontent.com/11189337/166852241-6760ef15-cabf-444f-a951-ab8316e90d48.png)

This will tell me if it's setting it correctly.

**Comment by Haplo064** [comment]
(For version 2.1.0.0, coming very soon)

**Comment by StarScythe** [comment]
![image](https://user-images.githubusercontent.com/104999536/166955993-14b6d801-0ed2-46e6-b9da-81eb7de824ef.png)
Here's what my debug logging says, colors and stuff work fine for the most part, but not on NPC bubbles, but scaling works on neither.

**Comment by aradiel** [comment]
![image](https://user-images.githubusercontent.com/22911153/166980225-70c3bada-b803-409f-a63f-a6a81abcda09.png)
![image](https://user-images.githubusercontent.com/22911153/166980699-eac50f1e-fd78-414c-aff1-ea4e2e4600a0.png)
My debug info, looks similar to StarScythe's

**Comment by Haplo064** [comment]
Hrmmmmmmm.

I don't know why it's not working, all signs point to it should be doing it.
I'll have to add in further debug info to track it down.

**Comment by aradiel** [comment]
![image](https://user-images.githubusercontent.com/22911153/169610818-6322fb29-0e19-4e17-8f5e-1966b5c68cd6.png)
Yup, can confirm it works with that setting.

though I think i'll pass on remaking my entire hud lol

**Comment by Grandorg22** [comment]
Hope we can get the fix with 6.2.
Also not really want to redo my Hud for the scaling to work. :)

**Comment by LalaGobbie** [comment]
Any updates on this issue?

**Comment by Noikar** [comment]
Yeah would be nice if this would be fixed because on 1440, turning the scaling up is way too big and I don't wanna have to reshrink everything to like 60% on high res mode just to make the bubbles readable.
